test/partitioning.jl

```diff
@@ -85,12 +85,27 @@
     p = partition(grid, BlockPartition(T(5),T(2)))
     @test length(p) == 12
     @test Set(nelements.(p)) == Set([5,10])
+
+    grid = CartesianGrid{T}(50, 50, 50)
+
+    p = partition(grid, BlockPartition(T(1.), T(1.), T(1.), neighbors = false))
+    @test length(p) == 125000
+    @test Set(nelements.(p)) == Set(1)
+    @test metadata(p) == Dict{Any,Any}()
+
+    p = partition(grid, BlockPartition(T(5.), T(5.), T(5.), neighbors = true))
+    @test length(p) == 1000
+    @test Set(nelements.(p)) == Set(125)
+    n = metadata(p)[:neighbors]
+    @test length(n) == length(p)
+    @test all(0 .< length.(n) .≤ 27) 
```

Comment:
```suggestion
    @test all(0 .< length.(n) .< 27) 
```
I think we can guarantee strict inequality since the center block is not included.